### DISCUSSION TITLE
Fractal guardians spawn in vents / other weird places in clear ruin mission

### DISCUSSION THREAD

**Discussion by SomeRandomNoobKekeke** [FakeFishGames/Barotrauma#13770] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

So they spawn stuck and helpless
Almost in every clear ruin mission in unstable at least one guardian is stuck
I couldn't reproduce in stable though, mb i'm just too lazy

![20240412110319_1](https://github.com/FakeFishGames/Barotrauma/assets/122838333/05144735-85c1-44df-8d8f-340685154d21)
![20240412121155_1](https://github.com/FakeFishGames/Barotrauma/assets/122838333/f2c19fec-d03b-4b17-a3a6-bdb7e2027824)
![20240412134339_1](https://github.com/FakeFishGames/Barotrauma/assets/122838333/c760bcf2-5388-439e-8679-0a4c2c5d8279)
![20240412134938_1](https://github.com/FakeFishGames/Barotrauma/assets/122838333/b030d8e6-cc68-46d3-9edf-b8bd3ad0e4cf)
![20240412135546_1](https://github.com/FakeFishGames/Barotrauma/assets/122838333/4b4fadde-88b3-4215-ab2d-4107e04e0ca2)
![20240412140428_1](https://github.com/FakeFishGames/Barotrauma/assets/122838333/20d1bcd6-3b7e-4bd1-b071-eac93a0b539e)
![20240412135953_1](https://github.com/FakeFishGames/Barotrauma/assets/122838333/45b305b4-f2c4-4018-9de5-438434df95dc)

### Reproduction steps

Start clear ruin mission on 100% difficulty
Find guards

### Bug prevalence

Happens every now and then

### Single player or multiplayer?

Single player

### Version

v1.4.0.0 (unstable)

### Which operating system did you encounter this bug on?

Windows

**Comment by SomeRandomNoobKekeke** [comment]
Hmm... seems to be magically fixed 
![barodev small](https://github.com/FakeFishGames/Barotrauma/assets/122838333/03d6c500-f4fb-4816-8826-0a86cd7cc6d0)
I can't find a single example in 1.4.3.0 while in 1.4.2.0 at least one was stuck in every mission
Also in 1.4.2.0 there always been 4 guards while in 1.4.3.0 there's 2-4 guards

**Comment by Regalis11** [comment]
Not fixed, there were no changes in 1.4.3.0 that could've affected this

**Comment by itchyOwl** [comment]
Converted into an issue in https://github.com/FakeFishGames/Barotrauma/issues/13779. Can be difficult to repro, but we think we know what causes it.

**Comment by Jasontti** [comment]
Closing as completed
